applications/nrf_desktop/README.rst

```diff
@@ -708,25 +827,37 @@ Precision Aim button
     * |nRF_Desktop_cancel_operation|
 
 Desktop mouse
-~~~~~~~~~~~~~
++++++++++++++
 
 The following predefined buttons are assigned to peer control operations for the desktop mouse:
 
 Scroll wheel button
     * Long-press to initialize and confirm the peer erase.
       The scroll must be pressed before the mouse is powered up.
 
-        * |nRF_Desktop_confirmation_effect|
+      .. figure:: /images/nrf_desktop_desktop_mouse_side.svg
+         :scale: 25 %
+         :alt: nRF Desktop desktop mouse - side view
 
+         nRF Desktop desktop mouse - side view
+
+     |nRF_Desktop_confirmation_effect|
     * |nRF_Desktop_cancel_operation|
 
 Keyboard
-~~~~~~~~
+++++++++
 
 The following predefined buttons or button combinations are assigned to peer control operations for the keyboard:
 
 Page Down key
     * The Page Down key must be pressed while keeping the Fn modifier key pressed.
+
+      .. figure:: /images/nrf_desktop_keyboard_top.svg
```

Comment:
@pdunaj , do we need LED1 on this image? I think we can keep it, but would have to add explanation in a bullet point below.